### PR TITLE
Autogreet addon

### PR DESCRIPTION
**Pull request by Touchcreator**
Auto greet.

**Resolves**

Having to type a greeting when on a scratcher's profile

**Changes**

I added a greet button.

**Reason for changes**

For those lazy people who don't like to type a greeting.

**Tests**
I have tested this. It works exactly how expected.

### CODE REVIEW

File: addons/pf-paste/addon.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      "name": "Touchcreator, retronbv & The Colaber"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/users/*"]
+    }
+  ],
+  "tags": ["community", "beta", "paste"],
```
Comment (YOYITsM3M8) [inline addons/pf-paste/addon.json:15]: you cannot make your own tags

Comment (Touchcreator) [reply addons/pf-paste/addon.json:15]: Ok

File: addons/pf-paste/addon.json [hunk 2]
```diff
@@ -12,6 +12,6 @@
       "matches": ["https://scratch.mit.edu/users/*"]
     }
   ],
-  "tags": ["community", "beta", "paste"],
```
Comment (YOYITsM3M8) [inline addons/pf-paste/addon.json]: keep these but not paste

Comment (Touchcreator) [reply addons/pf-paste/addon.json]: Done

File: addons/pf-paste/addon.json [hunk 3]
```diff
@@ -12,6 +12,6 @@
       "matches": ["https://scratch.mit.edu/users/*"]
     }
   ],
-  "tags": [],
+  "tags": ["community", "beta",],
```
Comment (YOYITsM3M8) [inline addons/pf-paste/addon.json:15]: that last comma is not needed and wil break the addon.json

**Comment by GrahamSH-LLK** [comment]
This is honestly a useless addon. If anyone has any objections, I'm denying this.

**Comment by apple502j** [comment]
pls no

**Pull request closed without merging**